code/modules/events/supermatter_surge.dm

```diff
@@ -4,18 +4,36 @@
 	weight = 15
 	max_occurrences = 4
 	earliest_start = 10 MINUTES
+	var/forced_power
+
+/datum/round_event_control/supermatter_surge/admin_setup()
+	if(!check_rights(R_FUN))
+		return
+
+	var/selected_power = input("How severe should it be? Between 1000 and 100000") as num|null
+	if(!selected_power)
+		selected_power = 1000
+	selected_power = clamp(selected_power, 1000, 100000)
```

Comment:
Why clamp it? The adminbus demands class -10000

Reply:
It's clamped, deal with it